interface/forms/eye_mag/js/eye_base.php

```diff
@@ -1221,14 +1220,14 @@ function build_DX_list(obj) {
                if (obj.PMSFH['POH'][key].diagnosis > '' ) {
                diagnosis = "<code class='float-right ICD_CODE'>"+obj.PMSFH['POH'][key].code+"</code>";
                }
-               out += "<li class='ui-widget-content'><span name='DX_POH_"+key+"' id='DX_POH_"+key+"'>"+obj.PMSFH['POH'][key].title+"</span> "+diagnosis+"</li>";
+               out += "<div class='droppable'><li class='ui-widget-content draggable'><span name='DX_POH_"+key+"' id='DX_POH_"+key+"'>"+obj.PMSFH['POH'][key].title+"</span> "+diagnosis+"</li></div>";
                });
         $.each(obj.PMSFH['POS'], function(key, value) {
                diagnosis='';
                if (obj.PMSFH['POS'][key].diagnosis > '' ) {
                diagnosis = "<code class='float-right ICD_CODE'>"+obj.PMSFH['POS'][key].code+"</code>";
                }
-               out += "<li class='ui-widget-content'><span name='DX_POS_"+key+"' id='DX_POS_"+key+"'>"+obj.PMSFH['POS'][key].title+"</span> "+diagnosis+"</li>";
+               out += "<div class='droppable'><li class='ui-widget-content draggable'><span name='DX_POS_"+key+"' id='DX_POS_"+key+"'>"+obj.PMSFH['POS'][key].title+"</span> "+diagnosis+"</li></div>";
                });
     }
     if ($('#inc_PMH').is(':checked') && obj.PMSFH['PMH']) {
```

Comment:
Also, I found this mainly javascript, is it possible to convert 
the file extension to js?

Reply:
I wouldn't pick that battle (ie. would be complicated)
There are places in this very large script using php and the hope for this eye form related code is to just fix what needs fixing and then move away as quickly as possible to work on something else; since this is really a very complicated module/form and could suck up a lot of time :)

Reply:
Ok, I got it.